rtcpeerconnection.go

```diff
@@ -27,6 +29,22 @@ import (
 // comparisons when no value was defined.
 const Unknown = iota
 
+// RTCPeerLogger is the default logging struct for rtcpeerconnection.
```

Comment:
I would move `RTCPeerLogger` + `webrtcLogger` to a dedicated package

This way other packages can import/use the logger (like `github.com/pions/internal/sctp`) right now we can't pass our logger instance down to other packages.

Reply:
Placed them in their own package pkg/logging/. The interface and struct names seem slightly unintuitive. Let me know if you have a better naming convention.